stdlib/turtle.pyi

```diff
@@ -381,6 +379,16 @@ if sys.version_info >= (3,):
     def onkeypress(fun: Callable[[], Any], key: Optional[str] = ...) -> None: ...
     onkeyrelease = onkey
 
+# Functions copied from _Screen:
+
+def setup(
+    width: Union[int, float] = ..., height: Union[int, float] = ..., startx: Optional[int] = ..., starty: Optional[int] = ...
+) -> None: ...
```

Comment:
```suggestion
def setup(width: float = ..., height: float = ..., startx: Optional[int] = ..., starty: Optional[int] = ...) -> None: ...
```

For type checking purposes, `int` is a sub-type of `float` (per PEP 484). So, `Union[int, float]` is equivalent to just `float`, at least in argument types.